examples/spark/pytorch/pytorch_lightning_spark_mnist.py

```diff
@@ -124,13 +125,54 @@ def validation_epoch_end(self, outputs):
             tensorboard_logs = {'val_loss': avg_loss}
             return {'avg_val_loss': avg_loss, 'log': tensorboard_logs}
 
-
     model = Net()
 
     # Train a Horovod Spark Estimator on the DataFrame
     backend = SparkBackend(num_proc=args.num_proc,
                            stdout=sys.stdout, stderr=sys.stderr,
                            prefix_output_with_timestamp=True)
+
+    from pytorch_lightning.callbacks import Callback
+
+    epochs = args.epochs
+
+    class MyDummyCallback(Callback):
+        def __init__(self):
+            self.epcoh_end_counter = 0
+            self.train_epcoh_end_counter = 0
+
+        def on_init_start(self, trainer):
+            print('Starting to init trainer!')
+
+        def on_init_end(self, trainer):
+            print('Trainer is initialized.')
+
+        def on_epoch_end(self, trainer, model):
+            print('A epoch ended.')
+            self.epcoh_end_counter += 1
+
+        def on_train_epoch_end(self, trainer, model, unused=None):
+            print('A train epoch ended.')
+            self.train_epcoh_end_counter += 1
+
+        def on_train_end(self, trainer, model):
+            print('Training ends')
+            assert self.epcoh_end_counter == 2 * epochs
+            assert self.train_epcoh_end_counter == epochs
+
+    callbacks = [MyDummyCallback()]
+
+    # FIXME: add EarlyStopping and ModelCheckpoint
```

Comment:
Maybe remove this for now? We don't want to confuse users who are just looking for an example of the API.

Reply:
Sure, remove them for now.